mobile/examples/object_detection/android/.idea/gradle.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do we need the `./idea` folder to be checked in?

normally they are added to .gitignore

Reply:
yeah, They can be generated at runtime, but we need to set the "configuration" once.